### PR TITLE
Update activations.py

### PR DESCRIPTION
**Pull request by darsh8200**
All spelling corrected in the documentation of the code.

### CODE REVIEW

**Comment by darsh8200** [comment]
@googlebot I signed it!

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def selu(x):
   and the number of inputs is "large enough"
   (see references for more information).
 
-  ![](https://cdn-images-1.medium.com/max/1600/1*m0e8lZU_Zrkh4ESfQkY2Pw.png)
+  https://cdn-images-1.medium.com/max/1600/1*m0e8lZU_Zrkh4ESfQkY2Pw.png
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: Don't remove the markdown markers here. They are useful to make the image show up as an image on the documentation page

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -380,7 +380,7 @@ def deserialize(name, custom_objects=None):
       x : String
 
   Returns:
-      Tensorlow Activation function denoted by input string.
+      Tensorflow Activation function denoted by input string.
```
Comment (mihaimaruseac) [inline tensorflow/python/keras/activations.py]: Capital `F` please

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/keras/activations.py (hunk 2 above)
Comment (darsh8200) [reply tensorflow/python/keras/activations.py]: Done and commited the changes.

**Review by rthadur — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**